brownie/exceptions.py

```diff
@@ -283,7 +283,7 @@ def decode_typed_error(data: str) -> str:
     selector = data[:10]
     if selector == "0x4e487b71":
         # special case, solidity compiler panics
-        error_code = int(data[4:].hex(), 16)
+        error_code = int(HexBytes(data[10:]).hex(), 16)
```

Comment:
I think this is a safer approach, in some cases the returned data is already formatted to `HexBytes`.  Not sure why it's inconsistent to begin with..

```suggestion
        error_code = int(HexBytes(data)[4:].hex(), 16)
```

Reply:
ahh wait i didn't zoom out enough...  your approach is correct